mods/d2k/rules/infantry.yaml

```diff
@@ -229,6 +229,22 @@ sardaukar:
 		EmptyWeapon: SardDeath
 		Chance: 100
 
+mpsardaukar:
```

Comment:
It seems that the Skirmish AI isn't able to build Sardaukars as Harkonnen due to https://github.com/OpenRA/OpenRA/blob/bleed/mods/d2k/rules/ai.yaml#L54. So it needs to be updated.